programs/dasheri/src/pool/instructions/create_mango_account.rs

```diff
@@ -6,9 +6,15 @@ use solana_program::program::invoke_signed;
 #[derive(Accounts)]
 #[instruction(account_num: u64, bump: u8)]
 pub struct PoolCreateMangoAccount<'info> {
+    // REVIEW: So this can be done for any mango program/group permissionlessly?
+    // There's probably no way to validate that mango_program is a real mango program...
```

Comment:
at least a program check could be done, I think only programs are marked as executable no?

Reply:
Yeah, but I meant that a sneaky attacker program could provide the API your program needs, but then run away with the funds. But it's up to the caller of this function here to enter the correct ids, should not be an issue